test/unit/storage/entities/account.js

```diff
@@ -34,16 +32,14 @@ describe('Account', () => {
 		it('should throw error for in-valid filters');
 		it('should accept only valid options');
 		it('should throw error for in-valid options');
-		it(
-			'should call adapter.executeFile with proper param for FIELD_SET_SIMPLE'
-		);
-		it('should call adapter.executeFile with proper param for FIELD_SET_FULL');
+		it('should call adapter.executeFile with proper param for extended=false');
+		it('should call adapter.executeFile with proper param for extended=true');
 		it('should accept "tx" as last parameter and pass to adapter.executeFile');
 		it(
-			'should resolve with one object matching specification of type definition for FIELD_SET_SIMPLE'
+			'should resolve with one object matching specification of type definition of simple object'
```

Comment:
This is more of an integration test as in unit we're not using the DB then we'll be testing a mock is correct.

Reply:
We will not be mocking for unit tests on DB level execution. We already have detail discussion on this topic. For DB execution, we will execute the queries directly on PostgreSQL and check the response. That's how we already doing and will use the same pattern.

Reply:
@lsilvs and I had detail discussion on this topic. He may add his thoughts here.

Reply:
Ohh as this were unit tests I thought we will not be hitting the DB at all. Shouldn't then move this tests to integration then?

Reply:
I definitely think we should have tests that execute the query and check the response. I don't mind how we call them but I do think they should be separated from these tests here.

Reply:
After discussing it was agreed for this tests to connect to the DB for executing queries in the cases that mocking is not enough to test a method so I'm resolving this.